static/index.html

```diff
@@ -1,11 +1,15 @@
 <!DOCTYPE html>
-<html lang="en">
-<head>
+  <html lang="en">
+   <head>
     
     <title>Vanilla Bean Todo list</title>
-    <script src= "index.js"></script>
-</head>
-<body>
+    
+  </head>
+  <body>
     <h1>Vanilla Bean Todo List!<h1>
```

Comment:
your missing the `/` on your closing tag, that's why everything looks like it's wrapped in an h1

Reply:
ohhh shitaki mushrooms